src/ColumnSchema.php

```diff
@@ -146,6 +149,11 @@ protected function phpTypecastValue(mixed $value): mixed
                     'f', 'false' => false,
                     default => (bool)$value,
                 };
+            case SchemaInterface::TYPE_BINARY:
+                if (is_string($value) && str_starts_with($value, '\\x')) {
+                    return hex2bin(substr($value, 2));
```

Comment:
Please cover tests this lines

Reply:
Test case for a default value covers the code.
Moved the code to the place where default values are casting.